datagrepper.spec

```diff
@@ -0,0 +1,278 @@
+# https://bugzilla.redhat.com/show_bug.cgi?id=955781
+%global pypi_name datagrepper
+%global pypi_version 1.0.0
+
+Name:           %{pypi_name}
+Version:        1.0.0
+Release:        5.20220513093908963333.spec.73.g3145ea8%{?dist}
```

Comment:
This seems odd. What's going to happen do this release value, will packit edit it? Then can we have a more neutral one instead in the repo? (like 1 or 0)?